bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/transport/MiIoAsyncCommunication.java

```diff
@@ -146,8 +146,12 @@ public int queueCommand(String command, String params)
             MiIoSendCommand sendCmd = new MiIoSendCommand(cmdId, MiIoCommand.getCommand(command),
                     fullCommand.toString());
             concurrentLinkedQueue.add(sendCmd);
+            String tokenText = Utils.getHex(token); // Obfuscate part of the token to allow sharing of the logfiles
+            tokenText = tokenText.substring(0, 8)
+                    .concat((tokenText.length() > 8) ? tokenText.substring(8, 24).replaceAll(".", "X") : "")
+                    .concat((tokenText.length() > 24) ? tokenText.substring(24) : "");
```

Comment:
It should not be smaller than 16 chars because of    ` private boolean tokenCheckPass(String tokenSting)` in the initialize, but I'll change it so it won't fail on <8 len